pkg/controller/config/config_controller.go

```diff
@@ -497,15 +498,32 @@ func (r *ReconcileConfig) applyCommunityResources(req reconcile.Request, cfg *op
 }
 
 func transformManifest(cfg *op.Config, m *mf.Manifest, addnTfrms ...mf.Transformer) (mf.Manifest, error) {
+	rbManifest := m.Filter(roleBinding)
+	rest := m.Filter(mf.Not(roleBinding))
 	tfs := []mf.Transformer{
 		mf.InjectOwner(cfg),
 		transform.InjectNamespaceConditional(flag.AnnotationPreserveNS, cfg.Spec.TargetNamespace),
-		transform.InjectNamespaceRoleBindingSubjects(cfg.Spec.TargetNamespace),
 		transform.InjectNamespaceCRDWebhookClientConfig(cfg.Spec.TargetNamespace),
 		transform.InjectDefaultSA(flag.DefaultSA),
 	}
+
 	tfs = append(tfs, addnTfrms...)
-	return m.Transform(tfs...)
+	rest, err := rest.Transform(tfs...)
+	if err != nil {
+		return *m, err
+	}
+
+	tfs = []mf.Transformer{
+		mf.InjectOwner(cfg),
+		transform.InjectNamespaceRoleBindingConditional(flag.AnnotationPreserveNS,
+			flag.AnnotationPreserveRBSubjectNS, cfg.Spec.TargetNamespace),
+		transform.InjectDefaultSA(flag.DefaultSA),
```

Comment:
do we need `InjectDefaultSA` while transforming RoleBindings

Reply:
No.